packages/core/test/__snapshots__/ctf.test.ts.snap

```diff
@@ -601,8 +591,7 @@ _createEffect(() => {
 `;
 
 exports[`ctf > $destructure > should transform $destructure bindings on $property 1`] = `
-"import { splitProps as _splitProps } from \\"solid-js\\";
-import { createEffect as _createEffect } from \\"solid-js\\";
+"import { createEffect as _createEffect, splitProps as _splitProps } from \\"solid-js\\";
```

Comment:
forgot this is fixed now in the new import helper

Reply:
I just ran `vitest -u` to fix the existing tests, I didn't fix it manually

Reply:
yeah I mean I haven't really run those for a long time.